### ISSUE TITLE
expand populated channels / or hide unpopulated channels (feature request)

### ISSUE THREAD

**Issue by Adorfer** [nunof07/discourse-mumble#8]
- expand channels "with users"
alternativly
- hide channels without users

on servers with many existing (permanent) channels it may be a challenge to look for "where the users are hiding", especially if there is a "deep tree". 

(i understand the role of the plusing as "show people if it's interesting to join")

**Comment by verymilan** [comment]
I totally agree with that - the count is nice but it's hard to find out where the heck the users are :D

**Comment by Adorfer** [comment]
I pachted my MumPI/classes/ChannelViewerProtocolProducer.php for that. 
(just to do the counting differently "without the afk" and "not showing unpopulated channels at all")
But i am not happy with that solution.
